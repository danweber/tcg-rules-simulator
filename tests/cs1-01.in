VERSION:2
# [Your Turn] [Once Per Turn] When this Monster attacks one of your opponent's Monster, this Monster gets +1000 DP for the turn.

FIELD1:Attacker,DW1-Vanilla,CS1-01 Other,DW1-Vanilla,CS1-01 Green,DW1-Birder
FIELD2:Vic1,CS1-02,REST Vic2,CS1-02,REST
SECURITY2:CS1-02
HAND1:DW1-Standup
GAME:P1 T2 MAIN 5
DUMP:DP
ECHO: -- Attack once with Attacker, have DP boost afterwards.
CMD:ATTACK Attacker Vic1
DUMP:DP
ECHO: -- Stand up Attacker, attack again, no DP boost this time
CMD:PLAY DW1-Standup
STEP:AUTOANSWER
CMD:ATTACK Attacker Vic2
DUMP:DP
ECHO: -- Attack into health
CMD:ATTACK Other Player
DUMP:DP
