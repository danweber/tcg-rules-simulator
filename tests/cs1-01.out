FIELD1:Attacker,DW1-Vanillamon,CS1-01,5000DP Other,DW1-Vanillamon,CS1-01,5000DP Green,DW1-Birder,3000DP
FIELD2:Vic1,CS1-02,REST,500DP Vic2,CS1-02,REST,500DP
GAME:P1 T2 MAIN 5
ECHO: -- Attack once with Attacker, have DP boost afterwards.
FIELD1:Attacker,DW1-Vanillamon,CS1-01,REST,6000DP Other,DW1-Vanillamon,CS1-01,5000DP Green,DW1-Birder,3000DP
FIELD2:Vic2,CS1-02,REST,500DP
GAME:P1 T2 MAIN 5
ECHO: -- Stand up Attacker, attack again, no DP boost this time
FIELD1:Attacker,DW1-Vanillamon,CS1-01,REST,6000DP Other,DW1-Vanillamon,CS1-01,5000DP Green,DW1-Birder,3000DP
FIELD2:
GAME:P1 T2 MAIN 1
ECHO: -- Attack into health
FIELD1:Attacker,DW1-Vanillamon,CS1-01,REST,6000DP Other,DW1-Vanillamon,CS1-01,REST,5000DP Green,DW1-Birder,3000DP
FIELD2:
GAME:P1 T2 MAIN 1
